webapp-django/crashstats/supersearch/views.py

```diff
@@ -240,7 +240,14 @@ def get_report_list_parameters(source):
     params = {}
 
     for key, value in source.items():
-        if not value:
+        if (
+            not value or (
+                # if value == [None]
+                isinstance(value, list) and
+                len(value) == 1 and
+                value[0] is None
+            )
+        ):
```

Comment:
Basically, why is it being sent `[None]` at all? Can we attack that problem first? 
I started looking at [these lines](https://github.com/AdrianGaudebert/socorro/blob/1144094-super-search-form-validation/webapp-django/crashstats/supersearch/views.py#L84-L86) to understand where it might come from but I'm no wiser.